redis/example/kustomization.yaml

```diff
@@ -2,14 +2,30 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests/dev-enablement/redis-shared?ref=v0.0.0
+  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests?ref=v0.0.0
+  #- ../manifests
 
 
-# Resource requests and limits
+# Add prefix to name
+namePrefix: service-cache-
+
+# OpsLevel annotations
+commonAnnotations:
+ "app.uw.systems/description": "cache for some service"
+ "app.uw.systems/tier": "tier_4"
+
 patches:
+  # replace namespace
+  - target:
+      namespace: change-me
+    patch: |-
+      - op: replace
```

Comment:
Can you just strip the namespace?

We repeatedly found omitting namespace in Kustomize bases makes them much easier to work with.

Reply:
[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/cdbefb7ffca7f2b270ceccd4482db53435030e0c#diff-29fa0c5dc8aa4d8d05f166a67bcef458ae148967740033b27d20f1313a3e43dcR22)

Reply:
Good idea, [removed namespace by default](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/4a5286be3f21d6c65c577e66d0e4e0303c261aa8)